wordpress-mobile/WordPress-Android
# Update media insertion to send caption information.

When using the block editor Insert captions when selecting a media item of the media library that has a caption defined.

To test:

 - Start the app
 - Start a new post in GB
 - Add an image block
 - Add an image of the WP media library that has a caption defined ( you can define it on the Media section of the site)
 - Check that the image is added and the caption is set

PR submission checklist:
